src/server/game/Spells/SpellEffects.cpp

```diff
@@ -3660,16 +3660,6 @@ void Spell::EffectScriptEffect(SpellEffIndex effIndex)
                     m_caster->CastSpell(unitTarget, 22682, true);
                     return;
                 }
-                // Decimate
-                case 28374:
-                case 54426:
```

Comment:
and this code shouldnt be there anyway, right? Spreaded code like this is a nightmare to maintain.